pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -832,7 +949,10 @@ let
       # Enable initrd support.
       BLK_DEV_INITRD    = yes;
 
-      PM_TRACE_RTC         = no; # Disable some expensive (?) features.
+      # Allows debugging systems that get stuck during suspend/resume
+      PM_TRACE             = yes;
+      PM_TRACE_RTC         = yes;
```

Comment:
Yes, that's a feature. The idea is that debugging information is stored in the RTC, and can be recovered if the machine is locked up. Systemd will revert the RTC to the correct value if the suspend/resume succeeds.

Reply:
> Yes, that's a feature. The idea is that debugging information is stored in the RTC, and can be recovered if the machine is locked up. Systemd will revert the RTC to the correct value if the suspend/resume succeeds.

That doesn't look like something that should be enabled by default, as it breaks completely https://github.com/NixOS/nixpkgs/pull/279459.

CONFIG_RTC_HCTOSYS
If you say yes here, the system time (wall clock) will be set using the value read from a specified RTC device. This is useful to avoid unnecessary fsck runs at boot time, and to network better.